new-examples/directories/directories.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE shows how to perform directory operations in Ballerina.
+keywords: ballerina, ballerina by examples, bbe, directory, file, path
```

Comment:
```suggestion
keywords: ballerina, ballerina by examples, bbe, directory, file, path
```
Please change this in all BBEs.

Reply:
Contained the related keywords of the BBE but all BBEs are related to the file module. So these may have the same keywords Can you check them?

Reply:
+1